reference/features/web_interface.md

```diff
@@ -1,6 +1,6 @@
 # Web interface
 
-After adding documents to your MeiliSearch, it is possible to try out the search engine with the integrated web interface. This only works in a [development environment](/reference/features/configuration.md#environment).
```

Comment:
Why did you remove this sentence? Did this behaviour recently change?

Reply:
We have mentioned this in the warning in line 8.